lectures/intro_supply_demand.md

```diff
@@ -23,6 +28,13 @@ Key infrastructure concepts that we'll encounter in this lecture are
 * social welfare as a sum of consumer and producer surpluses
 * competitive equilibrium
 
+We will use the following imports.
+
+```python
```

Comment:
@jstac these need to be `code-cell` to be executable.

Reply:
Thanks @mmcky .  Pushed changes.